app/screens/channel_info/__snapshots__/channel_info_header.test.js.snap

```diff
@@ -92,11 +92,8 @@ exports[`channel_info_header should match snapshot 1`] = `
       ]
     }
   >
-    <TouchableHighlight
-      activeOpacity={0.85}
-      delayPressOut={100}
+    <Component
       onLongPress={[Function]}
-      underlayColor="black"
```

Comment:
Strange that `channel_info_header.js` didn't change, but its rendering outcome has.

Reply:
☝️